airbyte-server/src/main/java/io/airbyte/server/CorsFilter.java

```diff
@@ -24,23 +24,29 @@
 
 package io.airbyte.server;
 
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerResponseContext;
 import javax.ws.rs.container.ContainerResponseFilter;
 
 // https://medium.com/@Leejjon_net/how-to-allow-cross-origin-requests-in-a-jax-rs-microservice-d2a6aa2df484
 public class CorsFilter implements ContainerResponseFilter {
 
-  private static final String ALLOW_ORIGIN = "Access-Control-Allow-Origin";
-  private static final String ALLOW_HEADERS = "Access-Control-Allow-Headers";
-  private static final String ALLOW_METHODS = "Access-Control-Allow-Methods";
+  public static final String ALLOW_ORIGIN = "Access-Control-Allow-Origin";
```

Comment:
Small question, as we already use `com.google.common.collect` package that is part of guava, isn't it better to use `com.google.common.net.HttpHeaders` here and use
`HttpHeaders.ACCESS_CONTROL_ALLOW_HEADERS` and etc instead?

Reply:
good idea.